modules/p2/ding_serendipity/css/ding_serendipity.css

```diff
@@ -21,6 +21,3 @@
   font-size: 160%;
   margin-bottom: 20px;
 }
-.pane-serendipity-ting-object .ting-object {
```

Comment:
Is it possible to override this in `_carousel.css` instead of manipulating this file?